commercetools/symfony-bundle/dev-reboot/config/packages/commercetools.yaml

```diff
@@ -0,0 +1,24 @@
+commercetools:
+    api:
+        default_client: 'default'
+        clients:
+            default:
+                client_id: '%env(CTP_CLIENT_ID)%'
+                client_secret: '%env(CTP_CLIENT_SECRET)%'
+                project: '%env(CTP_PROJECT_KEY)%'
+                scope: '%env(CTP_SCOPE)%'
+                oauth_url: '%env(CTP_AUTH_URL)%'
+                api_url: '%env(CTP_API_URL)%'
+    defaults:
```

Comment:
You should remove those config entries which defaults are handled by the Configuration